test/test_winevt.rb

```diff
@@ -41,6 +41,10 @@ def test_seek
       assert_true(@query.seek("first"))
       assert_true(@query.seek(:last))
       assert_true(@query.seek("last"))
+      assert_true(@query.seek(Winevt::EventLog::Query::Flag::RelativeToLast))
+      assert_true(@query.seek(Winevt::EventLog::Query::Flag::RelativeToFirst))
+      assert_true(@query.seek(Winevt::EventLog::Query::Flag::RelativeToFirst |
+                              Winevt::EventLog::Query::Flag::Strict))
```

Comment:
1 test should have only 1 assertion to get results of each flags.
Please consider adding data driven test.
In addition, please consider adding a test for a wrong flag like "123".

Reply:
Fixed in https://github.com/fluent-plugins-nursery/winevt_c/pull/8/commits/87cac4331e0772f5e86b287178680f5c219ec12c.